eng/pipelines/maintenance-packages.yml

```diff
@@ -2,8 +2,8 @@ trigger:
   batch: true
   branches:
     include:
-    - main
-    - release/*
+    - internal/main
+    - internal/release/*
```

Comment:
@carlossanlop - does this mean we won't have a nightly build that produces packages from maintenance-packages?  I'm not sure why we're removing those other branches instead of adding the internal paths.

Reply:
Similarly - you may want to include those same branches in the other definition - so that it can be used for rolling test builds on those branches.

IIUC - we use the branches include to represent which branches we deem that should be shipping.  Then we use darc to map those to channels to determine if they will publish to feeds or not.

Reply:
Asked around about this and yes it makes sense to have the two pairs of branches on both rolling builds. I was concerned there could be some kind of collision when publishing but the darc subscriptions will be tied to specific branches so there shouldn't be a problem.